crawl-ref/source/shopping.cc

```diff
@@ -298,7 +298,9 @@ unsigned int item_value(item_def item, bool ident)
             case SPMSL_PARALYSIS:
             case SPMSL_PENETRATION:
             case SPMSL_SILVER:
+#if TAG_MAJOR_VERSION == 34
             case SPMSL_STEEL:
+#endif
             case SPMSL_DISPERSAL:
```

Comment:
Please add the `#if TAG_MAJOR...` markers around the `SPMSL_POISONED` case as well. This also applies to the `switch-case`s in describe.cc and item-name.cc.